KarhooUISDK/Workers/Analytics.swift

```diff
@@ -102,27 +102,20 @@ final class KarhooAnalytics: Analytics {
             amount: String,
             currency: String
     ) {
-        let dateString: String
-        if #available(iOS 15.0, *) {
-            dateString = date.ISO8601Format()
-        } else {
-            let formatter = DateFormatter()
-            dateString = formatter.string(from: date)
-        }
         base.send(
                 eventName: .paymentFailed,
                 payload: [
                     Keys.message: message,
                     Keys.cardLast4Digits: last4Digits,
-                    Keys.date: dateString,
+                    Keys.date: date.toString(),
```

Comment:
Why we are sending date as a string? It will take different format depending on user locale, which will make this info impossible or extremely hard to compare on analytics side.

Reply:
Method updated. We ensure ISO8601 is always used